### DISCUSSION TITLE
Unable to level up new cards

### DISCUSSION THREAD

**Discussion by Momalpas** [MKT-Playbook/Hub#23] [General]
For last 2 or 3 tours I have been unable to add levels to new characters.  As an example Larry Wintertime has been available as a card in the app but I can’t increase the level to say I own him. Is there something I need to do? Using IPad and latest iOS

**Comment by TitusPosiadly** [comment]
Hey, thanks for reporting the issue.

Could you please add a bit more info by copying and filling out the template down below? Also please add a short screen capture, so I get a better understanding of the issue.

```
**Describe the bug**
_A clear and concise description of what the bug is._

**To Reproduce**
_Steps to reproduce the behavior:_
1. _Go to '...'_
2. _Click on '....'_
3. _Scroll down to '....'_
4. _See error_

**Expected behavior**
_A clear and concise description of what you expected to happen._

**Screenshots**
_If applicable, add screenshots to help explain your problem._

**Video**
_If applicable, upload (max 10MB or ~5 sec) or add a link to a video (e.g. YouTube or Vimeo) to help explain your problem._

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Version: [e.g. 1.1.8 (47)]
 - Jailbroken: [YES or NO]

**Additional context**
_Add any other context about the problem here._
```

**Comment by Momalpas** [comment]
Hi - thanks for replying!



Bizarrely when I went to the app today there appeared to be a background 
refresh of the database and them I could load the new characters so 
problem fixed!



Thanks for the app - it is fantastic



Lindsay



Sent via BT Email App



From: Titus Posiadly ***@***.***>

Sent: 13 January 2023 15:02:28 GMT

To: MKT-Playbook/Hub ***@***.***>

Cc: Momalpas ***@***.***>, Author 
***@***.***>

Subject: Re: [MKT-Playbook/Hub] Unable to level up new cards (Discussion 
#23)







Hey, thanks for reporting the issue.

Could you please add a bit more info by copying and filling out the 
template down below? Also please add a short screen capture, so I get a 
better understanding of the issue.
  **Describe the bug**
_A clear and concise description of what the bug is._

**To Reproduce**
_Steps to reproduce the behavior:_
1. _Go to '...'_
2. _Click on '....'_
3. _Scroll down to '....'_
4. _See error_

**Expected behavior**
_A clear and concise description of what you expected to happen._

**Screenshots**
_If applicable, add screenshots to help explain your problem._

**Video**
_If applicable, upload (max 10MB or ~5 sec) or add a link to a video 
(e.g. YouTube or Vimeo) to help explain your problem._

**Smartphone (please complete the following information):**
  - Device: [e.g. iPhone6]
  - OS: [e.g. iOS8.1]
  - Version: [e.g. 1.1.8 (47)]
  - Jailbroken: [YES or NO]

**Additional context**
_Add any other context about the problem here._


—
Reply to this email directly, view it on GitHub 
<https://github.com/MKT-Playbook/Hub/discussions/23#discussioncomment-4678729> 
, or unsubscribe 
<https://github.com/notifications/unsubscribe-auth/A5HNFDUVPLWOYQ2RDAX7CPLWSFVAJANCNFSM6AAAAAAT2H6XKY> 
.
You are receiving this because you authored the thread.Message ID: 
***@***.***>

**Reply by TitusPosiadly** [reply]
If that's the case, you were probably experiencing Issue #15. If you are ever experiencing this again please write a quick response that that thread (#15) and I will refresh the database again. If there is nothing else regarding this, I will close this discussion.
